code/game/objects/items/RCD.dm

```diff
@@ -75,6 +75,10 @@ RLD
 		loaded = loadwithsheets(W, plasmarglassmultiplier*sheetmultiplier, user) //8 matter for one plasma rglass sheet
 	else if(istype(W, /obj/item/stack/sheet/rglass))
 		loaded = loadwithsheets(W, rglassmultiplier*sheetmultiplier, user) //6 matter for one rglass sheet
+	else if(istype(W, /obj/item/stack/rods))
+		loaded = loadwithsheets(W, sheetmultiplier * 0.5, user) // 2 matter for 1 rod, as 2 rods are produced from 1 metal
+	else if(istype(W, /obj/item/stack/tile/plasteel))
```

Comment:
metal floor tiles are typed as plasteel?

Reply:
![yes](https://i.imgur.com/1doqcHN.png)

Reply:
whoever did this we are going to have a talk

Reply:
> whoever did this we are going to have a talk

https://github.com/tgstation/tgstation/commit/867b21c7ce839999c89ae80929eff4aebdcf3d79#diff-4ec5a6d8b25e1125fc4adec16395c0aaL118

it's been at least six years